### DISCUSSION TITLE
Flight Distance - Milestone 3

### DISCUSSION THREAD

**Discussion by Ganique** [Estia-advanced-programming/pandora-public#75] [Q&A] [answered]
# The units of data

I've already checked in the chat and in the wiki page but I found nothing. My question is : what are the units (units within the meaning of Physics) of these elements : maxAirSpeed, avgAirSpeed, maxEnginePower, avgEnginePower ? I need this information to calculate the flight distance of the aircraft.

Thank you in advance.

**Comment by dhmmasson** [comment, marked as answer]
https://github.com/Estia-advanced-programming/pandora-public/wiki/Pandora-%C2%BB-Instructions#constant-values
https://github.com/Estia-advanced-programming/pandora-public/wiki/Pandora-%C2%BB-Flight-Records#russian-fighter-jets-ru

airspeed : m/s
EnginePower: W
